src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -31,39 +31,44 @@
   /**
    * The target velocity of the NEO Brushless Motor.
    */
-  private final double shooterSetpoint = 1.0;
+  public final double shooterSetpoint = 1.0;
```

Comment:
Why did you make this public?  Member variables should *never* be public and should *always* be modified via an accessor (set) method.

Reply:
Done